lan867x/examples/lan867x_tcp_sockets/client/main/client.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
+#include <string.h>
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_netif.h"
+#include "esp_eth.h"
+#include "esp_event.h"
+#include "esp_log.h"
+#include "ethernet_init.h"
+#include "lwip/sockets.h"
+
+#define SOCKET_PORT         5000
```

Comment:
I'd probably also add `SOCKET_ADDRESS` as `"192.168.1.1"`

(or maybe a config option to the example?)